client.go

```diff
@@ -66,6 +66,16 @@ func (c *Client) Post(resource string, data url.Values) ([]byte, error) {
 	return c.do(req)
 }
 
+func (c *Client) Put(resource string, data url.Values) ([]byte, error) {
+	req, err := http.NewRequest("PUT", c.endpoint+resource, strings.NewReader(data.Encode()))
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Is there any endpoint that'd use different payload content-type, eg. JSON? Or is it always x-www-form-urlencoded?